src/inputs/TextFieldBase.tsx

```diff
@@ -56,6 +56,9 @@ export interface TextFieldBaseProps<X>
   hideErrorMessage?: boolean;
   // If set, the helper text will always be shown (usually we hide the helper text if read only)
   alwaysShowHelperText?: boolean;
+  // If set, the field will be treated as "read-only" in order to prevent the virtual keyboard on touch devices.
+  // It cannot be edited directly. It can be changed via other methods, though (i.e. DatePicker).
+  preventEdit?: boolean;
```

Comment:
Question: So is this different then our existing `readOnly` because the field still _looks_ editable?

Reply:
That's correct.